app/controllers/api/v0/base_controller.rb

```diff
@@ -46,11 +46,25 @@ def new_order
       end
 
       def submit_order
-        render json: SubmitOrder.new(params).process.to_hash
+        so_call do
+          so = ServiceCatalog::SubmitOrder.new(params.require(:order_id))
+          render :json => so.process.order
+        end
       end
 
       def fetch_plans_with_portfolio_item_id
-        render json: ServicePlans.new(params).process
+        so_call do
```

Comment:
`so`?  what is that?

Reply:
@Fryguy I wanted to create a wrapper function for all the service object (so) calls to handle all the exceptions in one place. Otherwise every function would have to rescue the exceptions.